src/btree/bt_handle.c

```diff
@@ -121,7 +121,6 @@ __wt_btree_open(WT_SESSION_IMPL *session, const char *op_cfg[])
 		F_SET(btree, WT_BTREE_READONLY);
 
 	/* Get the checkpoint information for this name/checkpoint pair. */
-	WT_CLEAR(ckpt);
```

Comment:
There were two callers of `__wt_meta_checkpoint()`, and both of them cleared the memory before the call, I thought it cleaner to have `__wt_meta_checkpoint()` do the clear instead of the callers.

Reply:
ack